csvcubed-pmd/tests/test-cases/README.txt

```diff
@@ -0,0 +1,4 @@
+The file "_ude_2.txt" and "bom-utf-16-be.srt" was used from https://github.com/chardet/chardet in the "tests/windows-1252/" and "tests/UTF-16/" directories repectively, for unit testing purposes.
+In this instance they are used to test the '_chardet' function in "/workspaces/csvwlib/csvcubed-pmd/csvcubedpmd/urireplacement/urireplacement.py", to test to see if the function detects 'Windows-1252' and "UTF-16" encoding types. 
```

Comment:
I wouldn't use a full file path here, because the path is likely to be different on the user's system depending on where they clone the repository to and whether they're using the docker container or not. I'd say the following would be fine: `function in "../../csvcubedpmd/urireplacement.urireplacement.py"` and (below) `The license for files ... is in ./LICENSE`. 

As long as it's clear that where the files are relative to this one, it's okay.